cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/index.tsx

```diff
@@ -38,18 +41,67 @@ export default class DeployedPipelineView extends React.PureComponent {
   }
 
   public render() {
-    return (
-      <Provider store={Store}>
-        <div className="pipeline-deployed-view pipeline-list-content">
-          <div className="deployed-header">
-            <PipelineCount />
-            <SearchBox />
-            <Pagination />
-          </div>
-
-          <PipelineTable />
-        </div>
-      </Provider>
-    );
+    return <DeployedPipelinesView />;
   }
 }
+
+const DeployedPipelinesView = () => (
+  <Query
+    query={gql`
+      {
+        applications {
+          name
+          artifact {
+            name
+          }
+          applicationDetail {
+            programs {
+              name
+              runs {
+                status
+                starting
+              }
+              ... on Workflow {
+                schedules {
+                  name
+                  status
+                  nextRuntimes
+                }
+              }
+            }
+          }
+        }
+      }
+    `}
+  >
+    {({ loading, error, data }) => {
+      if (loading) {
+        return <p>Loading...</p>;
+      }
+      if (error) {
+        return (
+          <p>
+            Error! `$
+            {error.message}`
```

Comment:
why newline?

Reply:
When I am running `yarn run cdap-dev-build-w`, it adds the new line between `$` and `{error.message}`. Should I remove the new line when I commit my changes?

Reply:
Done